facing "ginkgo: command not found" when following the contribution guide

```bash
➜  tidb-operator git:(master) ✗ make e2e-docker-push
GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS=linux GOARCH=amd64 ginkgo build tests/e2e
/bin/sh: ginkgo: command not found
make: *** [e2e-build] Error 127
```
I solved this by running `go get` ginkgo on my own. Maybe we should mention this instruction in the contribution guide or add it in retool.

--- a/docs/CONTRIBUTING.md
+++ b/docs/CONTRIBUTING.md
@@ -139,7 +139,7 @@ You can get the e2e test report from the log of testing pod:
 $ kubectl -n=tidb-operator-e2e logs -f tidb-operator-e2e
 ```
 
-To re-run e2e tests, delete the testing pod and apply it again:
+To re-run e2e tests, delete the testing pod and apply it again.
 
 ### Step 5: Keep your branch in sync
 

--- a/docs/CONTRIBUTING.md
+++ b/docs/CONTRIBUTING.md
@@ -96,6 +96,14 @@ $ git checkout -b myfeature
 
 ### Step 4: Develop
 
+#### Setup
+
+Make sure all the related tools are properly installed.
+
+```sh
+$ make check-setup
+```
+
 #### Edit the code
 
 You can now edit the code on the `myfeature` branch.
@@ -124,6 +132,14 @@ After Docker images are pushed to the DinD Docker registry, run e2e tests:
 ```sh
 $ kubectl apply -f manifests/tidb-operator-e2e.yaml
 ```
+
+You can get the e2e test report from the log of testing pod: 
+
+```sh
+$ kubectl -n=tidb-operator-e2e logs -f tidb-operator-e2e
+```
+
+To re-run e2e tests, delete the testing pod and apply it again:
 
 ### Step 5: Keep your branch in sync
 

